cpp/benchmarks/io/orc/orc_reader_multithreaded.cpp

```diff
@@ -112,13 +112,11 @@ void BM_orc_multithreaded_read_common(nvbench::state& state,
                    cudf::io::read_orc(read_opts, stream, rmm::mr::get_current_device_resource());
                  };
 
-                 threads.paused = true;
-                 for (size_t i = 0; i < num_files; ++i) {
-                   threads.submit(read_func, i);
-                 }
+                 threads.pause();
+                 threads.detach_sequence(0, num_files, read_func);
```

Comment:
This should be enough to make the `first_index` and `index_after_last` the same type
```suggestion
                 threads.detach_sequence(0ul, num_files, read_func);
```

Reply:
I went for the (more verbose) `decltype(num_files){0}`